### ISSUE TITLE
Fixes for multi-circle-membership

### ISSUE THREAD

**Issue by dottorer** [lale-help/lale-help#224]
Our pilot users in France are members in more than one circle. We have not fully supported / tested this in mvp1 so there are a few minor things coming up to fix:

**1. Circle switching** 
When logging in they should be taken only to their primary circle, but they need a way to switch between circles. I can see a simple way by offering this under the 'user menu' in the top nav bar as an option to switch to any of the other circles they are members of:

**_Helper Name_** 
* Your profile
* Switch to CIRCLE2_name
* Switch to CIRCLE3_name
* Log out

**2. Menu item 'Your profile' switches to a different circle**
When test user is in circle 4 doing work and then goes to 'Your Profile', the profile is opened but in the context of circle 5 now. 

**3. Verify admin right per circle**
Verify that user who is admin in circle A, but not in circle B does not gain admin rights while working in that circle.

**4. Ensure first circle remains primary**
When joining another circle, we need to ensure the user always remains in his first circle as primary. Currently when joining another circle, the user seems to get switched to the latest.

**5. Selection of primary circle**
On the helper profile, we used to have a drop down showing all circles a user was member of. It was labeled 'Primary circle' and determined which circle a user was taken to upon login. We decided to suppress / remove this for mvp1, but users are not requiring a way to set their primary circle. This value should only be changeable for users with more than 1 circle.

**Comment by dottorer** [comment]
@phil-monroe Is this a good an reliable way in your opinion to switch between circles? For now we suppressed the 'Find Circles' item in that user menu, but switching will be important for the French team.

**Comment by dottorer** [comment]
Assigning to @kristysnet to take a look at. This option should only appear for users that are members of more than 1 circle.

**Comment by dottorer** [comment]
Setting to prio 'high' since we encountered the bug where a used gets switched to the second circle as their primary once they join. 
@phil-monroe is this what the code does right now or was it a glitch somehow?

**Comment by dottorer** [comment]
@kristysnet - this feature is totally cool! @phillipoertel and I merged it in and tested it, looks spot on. Even works with the Join approval flow we implemented, just as we hoped!

Point 1,2,3 and 5 are totally working. Point 4 needs looking at. And here are a couple of nits so far:
1. When we have full circle names in the drop down menu, there isn't a lot of space. Is it possible to adjust the width of the drop down menu to avoid line breaks?
![image](https://cloud.githubusercontent.com/assets/14217285/14385197/64683d44-fd9f-11e5-9097-8d4402fe34a0.png)
2. When joining a new circle, this circle gets set as the new primary one. We should not do that but stick to the original one. The user can change that later if they desire.
3. On the user profile, primary circle can be selected in edit, but it is not shown in the view mode.

Well done!!!
